docs/r/versions.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<body><p><a href="../dev/r/">12.0.0.9000 (dev)</a></p>
+<p><a href="../r/">12.0.0 (release)</a></p>
+<p><a href="../11.0/r/">11.0.0.3 (release)</a></p>
```

Comment:
Should we remove " (release)" here?

```suggestion
<p><a href="../11.0/r/">11.0.0.3</a></p>
```